src/sentry/api/endpoints/group_tagkey_values.py

```diff
@@ -34,16 +35,10 @@ def get(self, request, group, key):
         """
         lookup_key = tagstore.prefix_reserved_key(key)
 
-        try:
-            environment_id = self._get_environment_id_from_request(
```

Comment:
Are we able to delete the `_get_environment_id_from_request` method?

Reply:
nope, still used in ~14 areas

Reply:
Darn.

Reply:
It seems like we could make a separate pass to remove the `_get_environment_id_from_request` method. Both the endpoint method and the environment helper function are very similar and we don't really need to have both, but that is a task for another day :smile: